Content.Tests/DMTests.cs

```diff
@@ -97,8 +99,11 @@ public void TestFiles(string sourceFile, DMTestFlags testFlags)
                     return DreamValue.Null;
                 }
             });
-
-            return (_dreamMan.DMExceptionCount == prev, result);
+            bool retSuccess = _dreamMan.LastException == prev;
+            if (retSuccess)
+                return (retSuccess, result, null);
+            else
+                return (false, result, _dreamMan.LastException);
```

Comment:
The main reason I didn't just have it be an infinite list is because this variable is also kept when a real server is running; in very expection-spamming scenarios it would just be a memory leak with no benefit to the runtime as of yet.

We can change this once try-catch is implemented and we have a stronger implementation for exception handling on the runtime.